anonymizer/OpenDiffix.Service/Program.fs

```diff
@@ -149,6 +148,8 @@ let handleExport
 
   File.WriteAllText(outputPath, output)
 
+  None
```

Comment:
if anyone is wondering - why use `Option` here? Here's a full story:

After a naive refactor, whereby I just returned "" here, which was later `printfn`ed, an obscure problem arose on the GUI end - file export would work but would show a popup about failed export.

The reason was that the JSON response being returned from here was `'\n'`, which isn't valid JSON from JSON.parse perspective, at the same time being a truthy JS value, so `json ? JSON.parse(json)` threw errors.

Reply:
Couldn't that have been solved easier by using `printf` or `Console.Out.Write` instead of `printfn`?

Reply:
no strong opinion here, the newline at the end seemed like a nice touch, in case anyone would like to fire these requests manually.

Let's change